### ISSUE TITLE
Doesn't work with NuxtJS

### ISSUE THREAD

**Issue by ChrisMcHarvey** [markdirish/node-odbc#135]
When running my program and it hits the line:
`const odbc = require('odbc')`
it returns the error 
![image](https://user-images.githubusercontent.com/14862980/96784966-22968000-13bc-11eb-807f-eefd7bcdf921.png)

When I run the suggested command to install the missing dependencies, `npm install --save child_process fs` it seems to install them fine but when I run my program again it returns the exact same error.

**Comment by markdirish** [comment]
Not sure how NuxtJS handles Node.js native modules... seems like it would still work.

Is there a Node.js process running under the covers?  If yes, what version is it? Something seems fouled up in your installation or loader, because `child_process` and `fs` are built-in Node.js modules, you should never have install them

**Comment by ChrisHarveyHub** [comment]
Ah that's interesting. I'm running node v12.16.1

**Comment by ChrisHarveyHub** [comment]
Yeah all the other modules I've install have worked fine but all the error messaging doesn't explain much. I just know the errors don't appear when I bypass odbc and run my db calls over the regular mysql db connection

**Comment by ChrisHarveyHub** [comment]
Just tried fully reinstalling node but got the same result on running. Can anyone help debug?

**Comment by ChrisHarveyHub** [comment]
Also seeing this error: 
![image](https://user-images.githubusercontent.com/14862980/96915812-b7f54b00-1474-11eb-8905-f87c51ef72d1.png)

**Comment by markdirish** [comment]
You should copy/paste the text starting at the very first error you see. The images are hard to parse (especially with the magenta background).

What it honestly looks like is that node-pre-gyp is trying to install (i.e. compile) when it hits the require the statement. In your `node_modules/odbc/lib` directory, is there a `napi-v{napi_build_version}` directory that has `odbc.node` inside of it?

**Comment by ChrisHarveyHub** [comment]
Apologies, below you'll find a full screenshot of the error.

As for the node-pre-gyp I had a hunch that was the culprit but wasn't sure where to begin. To your point, I see the odbc.node file inside node_modules/odbc/lib/bindings/napi-v3

![image](https://user-images.githubusercontent.com/14862980/96922088-67ceb680-147d-11eb-8264-e429e8b9e752.png)

**Comment by markdirish** [comment]
In `node_modules/odbc/lib/odbc.js`, change this line:

```
const bindingPath = binary.find(path.resolve(path.join(__dirname, '../package.json')));
```

To hardcode point to `./bindings/napi-v3/odbc.node` and see if that helps:

```
const bindingPath = './bindings/napi-v3/odbc.node';
```

**Comment by gianluca1606** [comment]
I think this Issue can be closed if you use normal imports it should work.
